config/cesm/machines/config_batch.xml

```diff
@@ -139,7 +139,7 @@
   </batch_system>
 
   <batch_system type="slurm" >
-    <batch_query>squeue</batch_query>
+    <batch_query per_job_arg="-j">squeue</batch_query>
```

Comment:
Yes, I tested on Sandia's slurm machine chama. The PBS machines are easier because they don't need the -j.